src/Illuminate/Routing/Route.php

```diff
@@ -534,6 +535,20 @@ public function domain()
     }
 
     /**
+     * Get the domain defined for the route.
+     *
+     * @return array|null
+     */
+    public function subdomain()
+    {
+        if (! isset($this->action['subdomain'])) {
+            return null;
+        }
+
+        return is_array($this->action['subdomain']) ? $this->action['subdomain'] : [$this->action['subdomain']];
```

Comment:
This can be simplified to `return (array) $this->action['subdomain'];` I believe